src/machine.adoc

```diff
@@ -178,7 +178,7 @@ User mode implemented +
 Vector extension +
 _Reserved_ +
 Non-standard extensions present +
-_Reserved_ +
+{cheri_base_ext_name} +
```

Comment:
Think we should have a note here that "if the Y bit is set then the implementation supports the RVY base ISA"

Reply:
yep - done